doc/src/vpr/debug_aids.rst

```diff
@@ -11,6 +11,9 @@ To access detailed echo files from VPR’s operation, use the command-line optio
 After parsing the netlist and architecture files, VPR dumps out an image of its internal data structures into echo files (typically ending in ``.echo``).
 These files can be examined to be sure that VPR is parsing the input files as you expect.
 
+To display and debug the placement move generator, enable the f_placer_flag in move_utils.cpp file. The new location of the moving block for each proposed move will be highlighted with GREEN and the old location will be highlighted with GOLD. The fanin and fanout blocks will also be highlighted. The move type, move outcome and delta cost will be printed in the status bar.
```

Comment:
Now that I understand f_placer_debug, I think this documentation is not correct/appropriate.  

"You can visualize and control the placement move generator whenever the placement engine is paused in the UI. 
Run with graphics enabled and set a breakpoint to stop placement at an appropriate point and then ... (now put in the info on move colors etc.)"

(Or else f_place_debug does something other than control breakpoints.  But if f_place_debug is being set automatically by placement breakpoints it doesn't make sense to tell users to go edit a file and set it to true.).

Reply:
resolved